docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+The output looks like so:
+```bash
+The client is able to save the authorization in a local file.
+This prevents the need to reinitialize the authorization for each transfer.
+You will be asked for this password whenever a new session is started.
+Please provide a new password to encrypt the local OSDF client configuration file:
+```
+Enter in a password and press `enter` to continue and you should see a response like so:
+```bash
+To approve credentials for this operation, please navigate to the following URL and approve the request:
+
+<https://some/link/here>
+```
+Pelican will display a URL in your terminal and indicate that you should visit the URL in your browser. After copying/pasting the URL to your browser, follow all the instructions there for logging in with CILogon. Once everything is successful, you should see a page like this:
+
+<ExportedImage src={"/pelican/cilogon.png"} alt={"Screenshot of CILogon success page"} />
+
+Finally, if the login is successful, Pelican will automatically fetch the token from the CILogon service and continue with the download.
+
+### For Issuers Without CILogon Support
+
+Some origins do not support authentication though CILogon. In this case, users must supply their own JWT that's signed by the origin.
+
+> We are working hard to document how to generate a token from protected origins, before it’s ready. If you have any questions or need help getting a token from the origin, please email us at [help@pelicanplatform.com](mailto:help@pelicanplatform.com) or open a discussion on our [GitHub discussions page](https://github.com/PelicanPlatform/pelican/discussions) and we are happy to assist you.
+
+Contact your federation administrator to get a token for downloading a protected object. For OSDF users, contact [support@osg-htc.org](mailto:support@osg-htc.org) for help.
+
+Once you have the token, copy and paste it into a file on your machine, and pass the path to the file containing the token to the Pelican client with `-t` flag.
+```bash
+# Pass the path to the file containing the token using -t flag
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> </local/path/to/file> -t </path/to/token/file>
+```
+
+For example, if a token is saved in a file named `my-token`, it can be used to get the object `/ospool/PROTECTED/auth-test.txt` by running:
+```bash
+# Get the object with the path to token file provided via -t flag
+$ pelican object get -f https://osg-htc.org /ospool/PROTECTED/auth-test.txt downloaded-auth-test.txt -t my-token
+```
+
+## PUT an object to Your Federation
+Another powerful Pelican client command is the `pelican object put` command. This command does a simple PUT request to add your object to your federation. Since the vast majority of federations will not allow public writes, we will assume for the example that we will need a token to perform these requests. Here is how you can use `pelican object put`:
```

Comment:
Public writes actually aren't possible in Pelican, so we can just say "Putting files into a federation requires a token"